blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+
+```
+
+TODO: Example config
+
+```
+
+<br />&nbsp;
+### Improved debugging experience
+
+A unified set of container metrics, compatible across both Kubernetes and Docker Compose deployment types,  provides the following information about containers through Grafana dashboards and alerting in both deployment types:
+
+ADD SCREENSHOTS - POSSIBLE TO STACK VERTICALLY TO IMPROVE READABILITY?
+
+Recent alerts history is now included when filing a bug report from the report-a-bug-page (for example, [https://sourcegraph.example.com/site-admin/report-bug](https://sourcegraph.example.com/site-admin/report-bug)) to better enable us to diagnose issues.
+
+We have added experimental support for detecting if your instance is over or under-provisioned through a set of new dashboards and warning-level alerts.
+
+
+## Sourcegraph admin features
+
+
+### Easier alerting configuration
+
+Easily configure Sourcegraph to send alerts about its health to [notifiers like Slack, PagerDuty, and webhooks]([https://docs.sourcegraph.com/admin/observability/alerting#setting-up-alerting](https://docs.sourcegraph.com/admin/observability/alerting#setting-up-alerting)). Set `observability.alerts` in your Sourcegraph configuration to automatically have alerts set up and subscribed to relevant notifiers in Grafana:
+
+```
+
+"observability.alerts": {
+
+  "id": "my-alert",
+
+  "level": "critical",
+
+  "notifier": { "type": "slack", /* ... */ }
+
+}
+
+```
+
+For more information, see [https://docs.sourcegraph.com/admin/observability/alerting](https://docs.sourcegraph.com/admin/observability/alerting).
+
+
+### Repository permission syncing on by default
+
+Permission syncing are now on by default for GitHub, Bitbucket Server, and GitLab for any integration that is using permissions. Documentation: [https://docs.sourcegraph.com/admin/repo/permissions#background-permissions-syncing](https://docs.sourcegraph.com/admin/repo/permissions#background-permissions-syncing)
+
+Background permissions syncing becomes the default method to sync permissions from code hosts. Please [read our documentation for things to keep in mind before upgrading](https://docs.sourcegraph.com/admin/repo/permissions#background-permissions-syncing). [#10972](https://github.com/sourcegraph/sourcegraph/pull/10972)
+
+
+### Notifications when Sourcegraph is out of date
+
+Sourcegraph now shows update notifications to site admins (at 1 month out-of-date), and then to users (at 4 months out-of-date). This will help keep your Sourcegraph instance current with the latest bug fixes and security updates, as well as new features and functionalities. This will also ensure that you don’t run into issues due to running on a stale version.
+
+These notifications start out as subtle informational alerts, and at 6+ months out-of-date, they will appear as red dismissible banners:
+
+SCREENSHOT
```

Comment:
@christinaforney do you also have the high-res version of this screenshot?

Reply:
Same as above, we will need to get this from @slimsag or @bobheadxi

Reply:
@slimsag @bobheadxi removing this placeholder text with a version of the screenshot. please send me the high-res version so i can update